README.markdown

```diff
@@ -39,6 +39,7 @@ Table of Contents
   - [Arch Linux](#arch-linux)
   - [AppImage](#appimage)
   - [Snap](#snap)
+  - [Conda-forge](#conda-forge)
```

Comment:
I added `conda-forge` at the end of the list of package manager options, but before Linux containers, to avoid having to make position selection choices, but if this should get moved to a different spot I'm happy to do so.